docs/stdlib/generic.rst

```diff
@@ -235,12 +245,10 @@ Generic
 
     :index: length count array
 
-    A polymorphic function to calculate a "length" of its first
-    argument.
+    Returns an :eql:type:`int64` of counted values within a type.
```

Comment:
```suggestion
    Returns a count of a given value's constituents.
```

- This doesn't return anything about a type but rather about a value _of_ that type.
- The type of the return value doesn't need to be in the one-liner.